gdcdictionary/schemas/follow_up.yaml

```diff
@@ -2251,6 +2264,8 @@ properties:
       - High Grade Dysplasia
       - High-grade Prostatic Intraepithelial Neoplasia (PIN)
       - HIV
+      - Human herpesvirus-6 (HHV-6)
+      - Human herpesvirus-8 (HHV-8)
```

Comment:
I think we might want to capitalize "herpesvirus" since it looks like we capitalize "Papillomavirus" and all enums seem to be title case.  I'll google around and change if appropriate.